modules/tinymce/src/plugins/help/main/ts/ui/KeyboardNavTab.ts

```diff
@@ -87,13 +87,16 @@ const description = `<h1>Begin keyboard navigation</h1>
 <p>Navigate between interactive components of this dialog tab by pressing <strong>Tab</strong> or <strong>Shift+Tab</strong>.</p>
 
 <p>Switch to another dialog tab by giving the tab menu focus and then pressing the appropriate <strong>Arrow</strong> key to cycle through the available tabs.</p>`;
-/* eslint-enable max-len */
 
-const tab = (): Dialog.TabSpec & { name: string } => {
+const langCode = I18n.getCode();
+
+const pLoadI18nDescription = (baseUrl: string): Promise<string> => Resource.load(`tinymce.html-i18n.help-keynav.${langCode}`, `${baseUrl}/js/i18n/keynav/${langCode}.js`);
```

Comment:
That's a good call, I added a handler to begin loading translations on editor `init`.
https://github.com/tinymce/tinymce/blob/17f7c36f368b92a87b1e9ea2d7fb8bb858919c72/modules/tinymce/src/plugins/help/main/ts/Plugin.ts#L26

`Resource.load` priortises retrieving from the cache by `id` but it would still need to be an asynchronous call since there is always a chance the resource has not yet been loaded.